source/Camera/Basic/QuickCaptureTutorial.md

```diff
@@ -0,0 +1,52 @@
+## Introduction
+
+This tutorial shows how few API calls are required to capture a point cloud with Zivid SDK.
+
+1. [Connect](#connect)
+2. [Capture](#capture)
+3. [Save](#save)
+
+### Prerequisites
+
+You should have installed Zivid SDK and C++ samples. For more details see [Instructions][installation-instructions-url].
+
+Before calling any of the APIs in the Zivid SDK, we have to start up the Zivid Application. This is done through a simple instantiation of the application ([go to source][start_app-url]).
+```csharp
+var zivid = new Zivid.NET.Application();
+```
+
+## Connect
+
+First we have to connect to the camera ([go to source][connect-url]).
+```csharp
+var camera = zivid.ConnectCamera();
+```
+
+## Capture
+
+Now we can capture a frame. The default capture is a single 3D point cloud ([go to source][capture-url]).
+```csharp
+var frame = camera.Capture();
+```
+
+## Save
+
+We can now save our results. By default the 3D point cloud is saved in Zivid format `.zdf` ([go to source][save-url]).
+```csharp
+var resultFile = "result.zdf";
+frame.Save(resultFile);
```

Comment:
Simplify to one line for this quick tutorial?

frame.Save("result.zdf");

Like in: https://github.com/zivid/python-samples/commit/4406342f5eae91e649fa393a8bb1bc9f253ae598

Reply:
You forgot this one. I pushed a commit for that now.